Mathlib/Geometry/Manifold/IsManifold.lean

```diff
@@ -1676,4 +1722,4 @@ instance : PathConnectedSpace (TangentSpace I x) := inferInstanceAs (PathConnect
 
 end Real
 
-set_option linter.style.longFile 1700
+set_option linter.style.longFile 1900
```

Comment:
Looking forward to seeing a splitting PR here sometime! :-)

Reply:
Good idea, but let's do that in a different PR :-)